.github/workflows/ci.yml

```diff
@@ -21,3 +21,16 @@ jobs:
 
     - name: Run tests
       run: yarn run test
+
+  coverage:
```

Comment:
How come you added an additional job instead of adding coverage to the previous step?

Reply:
Didn't want to change the previous job as codecov can be configured a lot to fail or not fail in certain cases. So I thought it'd be better to make it separately. Should I add it all in a single job?

Reply:
Yeah go ahead and add it to the first job. This file might need to be broken up, but we can do that separately.